packages/shylu/shylu_dd/frosch/src/Tools/FROSch_Tools_decl.hpp

```diff
@@ -239,6 +252,24 @@ namespace FROSch {
     template <class LO,class GO,class NO>
     RCP<const Map<LO,GO,NO> > BuildRepeatedMap(RCP<const CrsGraph<LO,GO,NO> > graph);
 
+    template <class LO,class GO,class NO>
+    Teuchos::RCP<Xpetra::Map<LO,GO,NO> > BuildMapFromNodeMap(Teuchos::RCP<const Xpetra::Map<LO,GO,NO> > &nodesMap,
+                                                           unsigned dofsPerNode,
+                                                           unsigned dofOrdering);
```

Comment:
Which map do you construct from the node map?

Reply:
The map with all degrees of freedom is constructed. Since the dofOrdering  on the coarse level is custom this function is unused and could be removed.

Reply:
Actually, I am wondering whether we already had a function for this before.

Reply:
Yes @searhein, you are right. This function should have already existed. There is no new code for this function in `FROSch_Tools_def.hpp`. Same goes for the function `BuildMapFromNodeMapRepeated` below.

Reply:
I removed the second declaration.